metrics/ignores/platform-android-vulkan.json

```diff
@@ -0,0 +1,4 @@
+{
+  "render-tests/fill-extrusion-color/function": "failing for Vulkan",
+  "render-tests/combinations/fill-opaque--image-translucent": "investigate"
```

Comment:
Could you also update this one to this issue https://github.com/maplibre/maplibre-native/issues/2829